ansible/roles/pdns/tasks/init_postgresql.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright (C) 2020 CipherMail B.V. <https://www.ciphermail.com/>
+# Copyright (C) 2020 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Create PostgreSQL database
+  postgresql_db:
+    name: '{{ pdns__postgresql_database }}'
+    owner: '{{ pdns__postgresql_role }}'
+    state: 'present'
+  delegate_to: '{{ pdns__postgresql_delegate_to }}'
+  register: pdns__register_postgresql_status
+
+- name: Import initial database schema
+  postgresql_db:
+    login_user: '{{ pdns__postgresql_role }}'
+    login_password: '{{ pdns__postgresql_password }}'
+    name: '{{ pdns__postgresql_database }}'
+    target: '{{ pdns__postgresql_schema }}'
+    state: 'restore'
+  when: pdns__register_postgresql_status is changed  # noqa no-handler
```

Comment:
This file looks like a job for `debops.postgresql`. I am not sure if schema import is supported, but the role could be extended.

Reply:
`debops.postgresql` doesn't support schema imports, sadly. I'll take a look at extending it, but it's a bit outside the scope of this PR so I'll open a new one. @drybjed can decide whether the `debops.pdns` role can be merged as-is or if we should do the `debops.postgresql` fix first.